City-of-Helsinki/berth-reservations
# VEN-530 | Add personal electricity option to pier

## Description :sparkles:
- Add a `personal electricity` options to pier to allow customers to bring their own contracts

## Issues :bug:
### Closes :no_good_woman:
**[VEN-530](https://helsinkisolutionoffice.atlassian.net/browse/VEN-530):** Add boolean field to pier to allow personal electricity contracts

## Testing :alembic:
### Automated tests :gear:️
```shell
$ pytest resources/tests/test_resources_queries.py::test_get_piers
```

### Manual testing :construction_worker_man:
1. Add a pier from the admin or the `createPier` mutation
2. You can pass the `personalElectricity` option from GraphQL or check the field on admin

## Screenshots :camera_flash:
**Pier node**
<img width="349" alt="image" src="https://user-images.githubusercontent.com/15201480/79451032-86e01d80-7fee-11ea-88d6-0139d76b7cc9.png">

**CreatePierMutationInput**
<img width="348" alt="image" src="https://user-images.githubusercontent.com/15201480/79451143-b8f17f80-7fee-11ea-96db-f8fa408a06bb.png">

## Additional notes 🗒️ 
The default value for the field is `False`
